src/components/animeDetail/Detaillist.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { IoArrowBackCircleOutline } from 'react-icons/io5';
+import Detail from './Detail';
+
+const Detaillist = ({ animeDetail }) => {
+  const { title } = animeDetail;
+
+  const styleUser = { color: 'white' };
+  return (
+    <section className="detailSection">
+      <div
+        className="containerSection"
+        style={{
+          backgroundImage: `url(${animeDetail.images.jpg.large_image_url})`,
+          backgroundSize: '5rem 5rem',
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'top 2rem left 5rem',
+        }}
```

Comment:
[OPTIONAL] 👍🏽 

- I'd like to kindly suggest that you write your styles separately (declare a variable and add the styles to it), just as you did on line 10, then, you can add it to your jsx. 🤗